signer/src/storage/model.rs

```diff
@@ -84,6 +91,17 @@ impl From<&crate::message::SweepTransactionInfo> for SweepTransaction {
     }
 }
 
+impl utxo::GetFees for Vec<SweepTransaction> {
+    /// Return the total fee of all the transactions in the vector.
+    fn get_fees(&self) -> Fees {
+        let total = self.iter().map(|tx| tx.fee).sum();
+        let total_size: u32 = self.iter().map(|tx| tx.vsize).sum();
```

Comment:
qq: do we risk to overflow when summing `vsize`s?

Reply:
In theory yes, but then we're up to 4MB worth of transaction data (well, roughly -- witness data is discounted to 1/4). So in practical terms, this should never happen.

But it's a good point, I'll refactor this a little to return a result, I guess it could potentially happen with `Fake`'d stuff.

Reply:
Refactored in [8018721](https://github.com/stacks-network/sbtc/pull/810/commits/8018721e7104fdbe5b368af70835ffbc23f437c5)